integration_tests/CMakeLists.txt

```diff
@@ -107,6 +107,40 @@ macro(RUN)
             if (${RUN_FAIL})
                set_tests_properties(${name} PROPERTIES WILL_FAIL TRUE)
             endif()
+        elseif(KIND STREQUAL "wasm")
+            # wasm test
+            # add_executable(${name} ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py ${RUN_EXTRAFILES})
+            # set_target_properties(${name} PROPERTIES LINKER_LANGUAGE Node)
+            add_executable(${name} ${name}.py ${RUN_EXTRAFILES})
+            target_compile_options(${name} PUBLIC --backend=wasm)
+            target_link_options(${name} PUBLIC --backend=wasm)
+            # set_target_properties(${name} PROPERTIES LINKER_LANGUAGE Python)
+            # add_custom_command(
+            #     OUTPUT ${name} ${name}.js
+            #     COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name}
+            #     DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py
+            #     VERBATIM)
+            # execute_process(COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name})
```

Comment:
I am trying to support `integration_tests` for the `wasm` backend. It seems there is a slight difference in the working of `integration_tests` in `lpython` as compared to `lfortran`. I tried adding the wasm testing code from `integration_tests/CMakeLists.txt` of `lfortran` and tried  to tweak it in several ways, but it still fails to work. Please, could someone possibly share what I might be missing?

The error, I am stuck at currently is
```bash
CMake Error: Cannot determine link language for target "expr_01".
CMake Error: CMake can not determine linker language for target: expr_01
```

I also tried to (simply) use `execute_process(COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name})` (`line 123` in the above code). It seems to work when only the tests related to `wasm` are executed from `integration_tests/run_tests.sh`. It fails with the following error when `wasm` is executed along with other backends in `integration_tests/run_tests.sh`.

```bash
make: *** No targets specified and no makefile found.  Stop.
```

Reply:
`execute_process(COMMAND lpython --backend wasm ${CMAKE_CURRENT_SOURCE_DIR}/${name}.py -o ${name})` seems  to work. I missed to leave the `integration_tests/b3` directory. I just pushed a commit [TEST: Fix: cd out from b3 directory](https://github.com/lcompilers/lpython/pull/1047/commits/6b8b793e1821ce93182e809c954ac815d15149b9) fixing it.